pkg/doc/verifiable/credential.go

```diff
@@ -426,6 +455,41 @@ func WithCredentialSchemaLoader(loader *CredentialSchemaLoader) CredentialOpt {
 	}
 }
 
+// WithJSONLDValidation uses the JSON LD parser for validation.
+func WithJSONLDValidation() CredentialOpt {
+	return func(opts *credentialOpts) {
+		opts.modelValidationMode = jsonldValidation
+	}
+}
+
+// WithBaseContextAndType validates that only the fields and values (when applicable) are present
```

Comment:
The other options have a suffix of "Validation" and this one doesn't?

Reply:
Fixed.